Demo/Test/Test.csproj

```diff
@@ -27,7 +27,18 @@
     <ProjectReference Include="..\Measures\Measures.csproj" />
   </ItemGroup>
 
-  <ItemGroup>
+  <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">

```

Comment:
Is this necessary? We'll never release the demo?

Reply:
Probably not, but this is just the standard stuff when creating a new project. We could take it out, but doesn't harm leaving it here